File: src/game/Assets/lol/BulletCase.cs
using System.Collections;
using System.Collections.Generic;
using UnityEngine;

public class BulletCase : MonoBehaviour {

    public Vector3 velocity;
    public float gravity = 0.1f;

    private float _turnAngle;
    private float _turnSpeed;

    // Use this for initialization
    void Start () {
        _turnAngle = Random.value * 360;
        _turnSpeed = Random.Range(-360.0f, 360.0f);
	}
	
	// Update is called once per frame
	void Update () {
        velocity.y -= gravity * Time.deltaTime;
        transform.position += velocity * Time.deltaTime;
        transform.localScale = Vector3.Lerp(transform.localScale, Vector3.zero, Time.deltaTime * 2.0f);
	}
}

File: src/game/Assets/lol/bulletCase.js
var life : float = 0.5;

private var destroyTime : float;
var velocity : Vector3;
private var gravity : float = 1.0f;
private var turnSpeed : float;
private var turnAngle : float;

function Start(){
	destroyTime = Time.time + life;
	turnAngle = Random.value * 360;
	turnSpeed = Random.Range(-360.0,360.0);
}

function Update () {
	if(Time.time > destroyTime){
		Destroy(gameObject);
	}
	velocity.y -= gravity * Time.deltaTime;
	transform.position += velocity * Time.deltaTime;
	transform.localScale = Vector3.Lerp(transform.localScale, Vector3.zero, Time.deltaTime * life * 2.0);
}